Description: Corrections to vncviewer/vncviewer.man
 Fix hyphens.
 Remove unsupported -user option.
Forwarded: no
Author: Jari Aalto <jari.aalto@cante.net>
Author: Sven Geuer <debmaint@g-e-u-e-r.de>
Last-Update: 2021-10-03
--- a/vncviewer/vncviewer.man
+++ b/vncviewer/vncviewer.man
@@ -88,11 +88,6 @@
 (bell) event. This option disables such behaviour
 (TightVNC\-specific).
 .TP
-\fB\-user\fR \fIusername\fR
-User name for Unix login authentication. Default is to use current
-Unix user name. If this option was given, the viewer will prefer Unix
-login authentication over the standard VNC authentication.
-.TP
 \fB\-passwd\fR \fIpasswd\-file\fR
 File from which to get the password (as generated by the
 \fBvncpasswd\fR(1) program). This option affects only the standard VNC
@@ -262,9 +257,6 @@
 .B passwordFile
 Equivalent of \fB\-passwd\fR option.
 .TP
-.B userLogin
-Equivalent of \fB\-user\fR option.
-.TP
 .B passwordDialog
 Whether to use a dialog box to get the password (true) or get it from
 the tty (false). Irrelevant if \fBpasswordFile\fR is set. Default
@@ -342,7 +334,7 @@
 \fBVNC_VIA_CMD\fR environment variable, expands patterns beginning
 with the "%" character, and executes result as a command assuming that
 it would create TCP tunnel that should be used for VNC connection. If
-not set, this environment variable defaults to "/usr/bin/ssh -f -L
+not set, this environment variable defaults to "/usr/bin/ssh \-f \-L
 %L:%H:%R %G sleep 20".
 
 The following patterns are recognized in the \fBVNC_VIA_CMD\fR (note
